combat-trainer.lic

```diff
@@ -654,8 +689,9 @@ class SpellProcess
     @before.each { |action| bput(action['message'], action['matches']) } if @before
 
     @custom_cast = "cast my #{game_state.weapon_name}" if game_state.casting_ignite
+    @custom_cast = "cast #{game_state.healing_target}" if game_state.casting_consume
```

Comment:
If I received a more severe wound in between the ritual phase and the cast phase, yea that would be. I think check_health would have to be called again though